packages/zarf-agent/manifests/deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         - name: private-registry
       containers:
         - name: server
-          image: "###ZARF_REGISTRY###/###ZARF_CONST_AGENT_IMAGE###"
+          image: "###ZARF_REGISTRY###/defenseunicorns/zarf/###ZARF_CONST_AGENT_IMAGE###"
```

Comment:
this seems hacky, but I get why it is happening.  Is there a way we could improve our regex matching for this in the future?

Reply:
So I actually thought about this a while and unfortunately no.  We include the host in the checksum by design and exclude it here intentionally.  The reason this is _hacky_ is because we aren't mutating it dynamically and it just so happens we previously could ignore the host on dockerhub. If anything we were more hacky before, it just wasn't as obvious.

Reply:
We could just do the tag, except that we use different images depending on if it's prod or dev.

Reply:
Yeah just kinda sucks